src/orion/algo/random.py

```diff
@@ -55,17 +55,15 @@ def suggest(self, num=1):
         .. note:: New parameters must be compliant with the problem's domain
            `orion.algo.space.Space`.
         """
+        if num is None:
+            num = 1
```

Comment:
should just go with `max_trials` like `gridsearch`?

Reply:
It would depend on what we decide here: https://github.com/Epistimio/orion/pull/604#discussion_r625873582

Reply:
So, based on f6aea11 there would be no default `num` anymore. Typically the producer would request `max_trials` to the random search at the beginning which would return the whole set of trials to run.